pom.xml

```diff
@@ -9,52 +9,36 @@
     <version>6</version>
   </parent>
 
+  <groupId>com.extendaretail</groupId>
   <artifactId>webcam-capture-parent</artifactId>
   <version>0.3.13-SNAPSHOT</version>
   <packaging>pom</packaging>
 
   <name>Webcam Capture Parent POM</name>
   <description>This is Webcam Capture root POM. Keywords: cam, webcam, camera, spycam, capture</description>
-  <url>https://github.com/sarxos/webcam-capture</url>
+  <url>https://github.com/extenda/webcam-capture</url>
   <inceptionYear>2012</inceptionYear>
 
   <organization>
-    <name>Bartosz Firyn (SarXos)</name>
-    <url>https://github.com/sarxos</url>
+    <name>Extenda Retail</name>
+    <url>https://github.com/extenda/webcam-capture</url>
   </organization>
 
-  <developers>
```

Comment:
Should we keep this one for attribution?

Reply:
Yes you're right, I was a actually a bit hestistant there what would be most correct. I'll bring it back